help/view-and-edit-your-message-drafts.md

```diff
@@ -14,24 +15,76 @@ so that you never lose your work. Drafts are saved for 30 days.
 
 {tab|desktop-web}
 
-1. Simply close the compose box. You can hit <kbd>Esc</kbd>, click
-the <i class="fa fa-remove"></i> in the top right corner of the
-compose box, or click on an empty part of the app.
+{!start-composing.md!}
+
+1. Write a message.
```

Comment:
This feels a bit too ambiguous to me -- "Type at least 3 characters of text into the compose box." might be best -- to document the fact that we won't save <=2 characters as a draft.